.drone.yml

```diff
@@ -0,0 +1,42 @@
+pipeline:
+  build_4:
+    image: node:4.4.2
+    commands:
+      - npm --loglevel warn install -g npm@3
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_5:
+    image: node:5
+    commands:
+      - npm --loglevel warn install
+      - npm run test:ci
+
+  build_app:
+    image: docker:1.11
+    environment:
+      - DOCKER_HOST=tcp://127.0.0.1:2375
+    commands:
+      - docker build -t rotm .
```

Comment:
Please add:

```
when:
      event: pull_request
```
as per https://github.com/UKHomeOffice/brp_app/blob/master/.drone.yml

This means we don't waste time building docker images for pushes just for PR's